docs/design-proposals/apiserver/openstack-api-support.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Support Openstack Compute APIs in Arktos
+authors:
+  - "@yb01"
+---
+
+# Support Openstack Compute APIs in Arktos
+## Background and context
+To boost Arktos VM adoption and to let Openstack users to easily explore Arktos VM support without a lot of changes from the users’ applications, it is decided to add a new set of APIs in Arktos to match Openstack Server related operations. Besides better user experiences, this new APIs will also help existing Openstack perf tests to be relatively easily converted to run against Arktos.
+
+## Goal and non-goals
+
+Goals:
+
+1. New set of REST interfaces in Arktos to support Openstack requests for Server CRUD
+1. New set of REST interfaces in Arktos to support Openstack requests for Server Actions
+1. Nova or Nova test tool can switch to Arktos for basic VM operation, by switching to the Arktos URL
```

Comment:
👍  Exactly. If we could reuse existing perf testing tool, that would save a lot of time.

Reply:
this is the goal !